perun/postprocess/regressogram/methods.py

```diff
@@ -0,0 +1,135 @@
+"""
+Module with regressogram computational method and auxiliary methods at executing of this method.
+"""
+
+import numpy as np
+import numpy.lib.function_base as np_bin_selectors
+from click import BadParameter
+from scipy import stats
+from sklearn import metrics
+
+from perun.postprocess.regression_analysis.tools import validate_dictionary_keys
```

Comment:
Fix: same as above: instead `import perun.postprocess.regression_analysis.tools as tools` and use qualified names.